Dockerfile

```diff
@@ -27,6 +24,12 @@ RUN chmod u+x /var/lib/neuro/entrypoint.sh
 WORKDIR /root
 ENV PATH=/root/.local/bin:$PATH
 
+
+ARG NEURO_EXTRAS_VERSION
+ARG NEURO_EXTRAS_PACKAGE="neuro-extras==$NEURO_EXTRAS_VERSION"
+RUN pip install --user $NEURO_EXTRAS_PACKAGE
```

Comment:
at the end so that to cache other layers. This part will change a lot since we'll be testing the image with different neruo-extras versions.